### PR TITLE
Move jquery

### PR DESCRIPTION
**Pull request by DmytroOrlov**

### CODE REVIEW

File: app/dal/CassandraCluster.scala [hunk 1]
```diff
@@ -25,11 +28,22 @@ class CassandraCluster @Inject()(conf: CassandraConfig, appLifecycle: Applicatio
...
+      import scala.collection.JavaConversions._
+      config.getStringList("devgym.db.cassandra.hosts")
+    }
+    def docker: Option[Boolean] = configuration.getBoolean("devgym.db.cassandra.docker")
+    def ip() = docker match {
+      case Some(true) => Try(Seq("docker-machine", "ip", "default").!!.trim).toOption
+      case _ => None
+    }
+
+    ip().fold(hosts) { ip => Seq(ip) }
```
Comment (novakov-alexey) [inline app/dal/CassandraCluster.scala]: What we are doing here? Configuration of cassandra is so complex? usage of fold function. Looks scary

File: app/models/Task.scala [hunk 2]
```diff
@@ -0,0 +1,9 @@
+package models
+
+case class Task(`type`: TaskType.TaskType, taskDescription: String, solutionTemplate: String, referenceSolution: String)
```
Comment (novakov-alexey) [inline app/models/Task.scala]: Maybe we mix all model classes into one file ?

Comment (DmytroOrlov) [inline app/models/Task.scala]: yep

File: app/dal/CassandraCluster.scala (hunk 1 above)
Comment (DmytroOrlov) [inline app/dal/CassandraCluster.scala]: Fold is common functional operation.

Here is short protocol:
Lets calculate hosts!
Try to figure out ip somehow. If failed use hosts from config else use discovered ip.
Now about ip: if we can try docker, let's try it.

Comment (novakov-alexey) [inline app/dal/CassandraCluster.scala]: Ok. Need to get used to this

File: build.sbt [hunk 3]
```diff
@@ -15,6 +15,8 @@ lazy val root = (project in file("."))
 
 lazy val runtimeSettings = Seq(
   libraryDependencies ++= Seq(
+    "org.webjars" % "jquery" % "2.2.0",
```
Comment (novakov-alexey) [inline build.sbt]: wow, what do we get with this web jar now?

Comment (DmytroOrlov) [inline build.sbt]: I have dropped jquery as file, but replaced with dependency. Is it ok?

Comment (novakov-alexey) [inline build.sbt]: I still can see jquery as file in the project

Comment (DmytroOrlov) [inline build.sbt]: 100% removed

**Pull request merged**